hive-metastore/src/test/java/org/apache/iceberg/hive/HiveTableTest.java

```diff
@@ -95,23 +95,24 @@ public void testCreate() throws TException {
 
     // check parameters are in expected state
     Map<String, String> parameters = table.getParameters();
-    Assert.assertNotNull(parameters);
-    Assert.assertTrue(ICEBERG_TABLE_TYPE_VALUE.equalsIgnoreCase(parameters.get(TABLE_TYPE_PROP)));
-    Assert.assertTrue("EXTERNAL_TABLE".equalsIgnoreCase(table.getTableType()));
+    Assertions.assertThat(parameters).isNotNull();
```

Comment:
assertThat can be statically imported here as well, since there's no previous usage in this file that would cause a large diff

Reply:
Added